src/pages/validator/amplifier/error-messages.mdx

```diff
@@ -0,0 +1,23 @@
+# Common Amplifier Error Messages
+
+The following are some common error messages encountered when working with verifiers and relayers on the Amplifier.
+
+## Verifiers
+
+- `command not found: axelard` — The symlink or alias to `axelard` is not working. Use full binary name instead for commands, such as `./axelard` version instead of `axelard version`.
+
+- `command not found: tofnd` — The symlink or alias to `tofnd` is not working. Use full binary name instead for commands, such as `tofnd-darwin-[version] --help` instead of `tofnd --help`.
```

Comment:
```suggestion
- `command not found: tofnd` — The symlink or alias to `tofnd` is not working. Use full binary name instead for commands, such as `tofnd-darwin-[version] --help` instead of `tofnd --help`.  Alternatively you can fix your alias, `PATH`, or symlink.
```